react/react2/week1-exercise/app/blogs/[slug]/page.jsx

```diff
@@ -0,0 +1,17 @@
+"use client";
+import { usePathname } from "next/navigation";
+import { useRouter } from "next/router";
+
+function BlogCard() {
+  const pathname = usePathname();
+  const slug = pathname.split("/").pop().replace(/-/g, " ").toUpperCase();
+
+  return (
+    <div>
+      <p className="mb-4 italic">Current Pathname: {pathname}</p>
+      <p className="mb-4 text-xl">Blog: {slug}</p>
```

Comment:
This is a solution, another one is using [the params prop](https://nextjs.org/docs/app/building-your-application/routing/dynamic-routes)